typedoc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "inputFiles": ["src/*"],
+    "exclude": ["**/index.js", "src/*/*/*", "src/globals.js", "src/registerServiceWorker.js"],
+    "mode": "modules",
+    "out": "docs"
+}
```

Comment:
Do you make sure you have the [EditorConfig](https://editorconfig.org/) plugin set up correctly in your editor? The [`.editorconfig`](https://github.com/cybersemics/em/blob/dev/.editorconfig) should ensure that all files end with a newline character.

Reply:
@pushkov-fedor

Reply:
Sorry for not responding, this page is getting massive and messy. 
I forgot to enable plugin in vscode, now it works :)